### DISCUSSION TITLE
user-rule.txt是否可以支持使用在线txt？

### DISCUSSION THREAD

**Discussion by wyfang** [shadowsocks/shadowsocks-windows#3274] [Ideas]
事实上当自己需要在多个设备上时候时，我希望user-rule可以使用线上文件。

**Comment by skysun210** [comment]
把shadowsocks放在类似onedrive的同步盘上，应该可以吧。

**Reply by wyfang** [reply]
> 把shadowsocks放在类似onedrive的同步盘上，应该可以吧。

是可以，不过我更需要的是类似这个规则在不同用户上共享的意思。
